src/components/CryptobotCard/index.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import model from 'src/images/Col-1.png';
+
+function CryptobotCard({ bot }) {
+
+  return (
+    <div className="bg-base-800  font-mulish h-full border-2 border-base-400 rounded-lg overflow-hidden">
+      <div>
+      {/* TODO: convert uri bytes & load custom 3d bots */}
+        <model-viewer
+          camera-controls
+          alt="3D Cryptobot"
+
+          src={bot.uri}
```

Comment:
@manangouhari  Can you help debug this. 

I am fetching 3d model uri from the contract(Contract link:
https://better-call.dev/delphinet/KT1Hsfoab2CmqRxBicYEq9CijizAPMVCpQBZ/storage). Now, this works If I pasted the string for eg:  h t t p s : / / c l o u d f l a r e - i p f s . c o m / i p f s / Q m S m X X q R B T R r H W g 7 o e P 2 n r v F w P a W 4 w c R J 8 N t F q T z 2 5 L e 8 o

But, when fetched from better-call dev it doesn't work. It's some wired bytes to string conversion bug.

Reply:
@manangouhari  If you aren't able to find a solution. Don't worry. I will just tweak the contract so that it accepts string rather than bytes.

Reply:
It's not worth the effort, I just change bytes data type to string. This issue is resolve now.

Reply:
Great. I was looking into it, then came back on this thread to double check the details. Glad it's resolved.